miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniappSdkInitializer.kt

```diff
@@ -76,9 +70,8 @@ class MiniappSdkInitializer : ContentProvider() {
                 baseUrl = manifestConfig.baseUrl(),
                 rasAppId = manifestConfig.rasAppId(),
                 subscriptionKey = manifestConfig.subscriptionKey(),
-                hostAppVersionId = manifestConfig.hostAppVersion(),
                 hostAppUserAgentInfo = manifestConfig.hostAppUserAgentInfo(),
-                isTestMode = manifestConfig.isPreviewMode() || manifestConfig.isTestMode()
```

Comment:
Thanks for fixing the conflicts, at this area, I think we kept [this line in master branch](https://github.com/rakutentech/android-miniapp/blob/e7dbff0de08ae436102e03752bc6d2d50c00eda0/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniappSdkInitializer.kt#L81).

Reply:
why do we use `isTestMode` instead `isPreviewMode`?

Reply:
I just checked that there is no difference between:
```
`isTestMode = manifestConfig.isPreviewMode() || manifestConfig.isTestMode()` 

// or

`isPreviewMode = manifestConfig.isPreviewMode() || manifestConfig.isTestMode()`. 
```

But, both are breaking changes when the HostApp _(by mistake)_ define both in `gradle.properties` configuration. 
I just checked this part, actually the more appropriate solution I think:

```
       MiniApp.init(
            context = context,
            miniAppSdkConfig = MiniAppSdkConfig(
                ...
              isTestMode = manifestConfig.isTestMode(),
              isPreviewMode = manifestConfig.isPreviewMode()
            )
        )
```

We can merge this PR as it is, then I'll investigate it and make a separate PR if it needs the new solution.

@minh-rakuten @corycaywood What do you think?

Reply:
This part is still using the primary constructor of `MiniAppSdkConfig`, so `isPreviewMode` was appropriate to use in this case.
Minh was right for the change.

Sorry, Project ID PR wasn't merged yet which is using the secondary constructor for backward compatibility. So, confusion arises.